monkey/monkey_island/cc/services/post_breach_files.py

```diff
@@ -12,8 +12,10 @@
 PBA_WINDOWS_FILENAME_PATH = ['monkey', 'post_breach', 'PBA_windows_filename']
 PBA_LINUX_FILENAME_PATH = ['monkey', 'post_breach', 'PBA_linux_filename']
 UPLOADS_DIR_NAME = 'userUploads'
-PBA_UPLOAD_PATH = ['monkey_island', 'cc', UPLOADS_DIR_NAME]
-UPLOADS_DIR = Path(*PBA_UPLOAD_PATH)
+
+
+_PBA_UPLOAD_PATH = [os.getcwd(), 'monkey_island', 'cc', UPLOADS_DIR_NAME]
+ABS_UPLOAD_PATH = Path(*_PBA_UPLOAD_PATH)
```

Comment:
How about this?
```python3
from monkey_island.cc.consts import MONKEY_ISLAND_ABS_PATH

_PBA_UPLOAD_PATH = [MONKEY_ISLAND_ABS_PATH, 'cc', UPLOADS_DIR_NAME]
ABS_UPLOAD_PATH = Path(*_PBA_UPLOAD_PATH)
```

Reply:
+1

Reply:
Can we shorten it down even more? I don't see `_PBA_UPLOAD_PATH` used anywhere else.

```python
from monkey_island.cc.consts import MONKEY_ISLAND_ABS_PATH

ABS_UPLOAD_PATH = Path(*[MONKEY_ISLAND_ABS_PATH, 'cc', UPLOADS_DIR_NAME])
```